.nvmrc

```diff
@@ -1 +1 @@
-16
+18
```

Comment:
Maybe align it with what we have as a Env variable in vercel?
Not sure which overwrites the other tho -

<img width="936" alt="Screenshot 2023-07-26 at 4 39 19 PM" src="https://github.com/temporalio/documentation/assets/34380806/b0049ae5-a5a8-4402-ad53-a914233163df">

Reply:
nvm - this keeps us on the latest version of 18 - nice.